onnxruntime/core/providers/vitisai/imp/graph.cc

```diff
@@ -107,12 +107,11 @@ void graph_save(const Graph& graph, const std::string& filename, const std::stri
   auto graph_proto_subgraph = graph.ToGraphProto();
   *model_proto->mutable_graph() = *graph_proto_subgraph;
   auto& logger = logging::LoggingManager::DefaultLogger();
-  auto filename_data_relative_path = std::filesystem::path();
   auto model = Model::Create(std::move(*model_proto), ToPathString(filename), nullptr, logger);
   if (initializer_size_threshold == std::numeric_limits<size_t>::max()) {
     model_proto = model->ToProto();
   } else {
-    model_proto = model->ToGraphProtoWithExternalInitializers(filename_dat, graph.ModelPath(), initializer_size_threshold);
+    model_proto = model->ToGraphProtoWithExternalInitializers(filename_dat, ToPathString(filename), initializer_size_threshold);
```

Comment:
The first and the second parameters of the ToGraphProtoWithExternalInitializers function has the same type. 
So, here you also need to call ToPathString for filename_dat. This is based on an assumption that `filename_dat` and `filename` are encoded in UTF-8.

Reply:
Hi, I have changed everything you requested. Could you take a look. @snnn